etcdmain/etcd.go

```diff
@@ -616,6 +617,42 @@ func setupLogging(cfg *config) {
 		}
 		repoLog.SetLogLevel(settings)
 	}
+	switch cfg.LogOutput {
+	case "", "stderr":
+		break
+
+	case "stdout":
+		capnslog.SetFormatter(capnslog.NewPrettyFormatter(os.Stdout, cfg.Debug))
+
+	case "journald":
+		f, err := capnslog.NewJournaldFormatter()
+		if err != nil {
+			plog.Warningf("couldn't log to journald (%v)", err)
+			return
+		}
+		capnslog.SetFormatter(f)
+
+	default:
+		if filepath.Ext(cfg.LogOutput) == ".log" {
```

Comment:
don't bother checking for .log? not sure if silently failing into stderr is the right behavior here